specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/AvailableWorkloadProfiles_Get.json

```diff
@@ -124,6 +124,19 @@
             "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/providers/Microsoft.App/availableManagedEnvironmentsWorkloadProfileTypes/CPUOptimized-3",
             "name": "CPUOptimized-3",
             "type": "Microsoft.App/availableManagedEnvironmentsWorkloadProfileTypes"
+          },
+          {
+            "location": "East US",
+            "properties": {
+              "displayName": "Consumption",
+              "billingMeterCategory": "ConsumptionBasedBilling",
+              "applicability": "Custom",
```

Comment:
how applicability is used by customer?

Reply:
This api is targeted to our portal team in order to select default workload profile to show in dropdown